src/otx/v2/adapters/torch/mmengine/mmaction/modules/models/backbones/movinet.py

```diff
@@ -0,0 +1,780 @@
+            padding (Union[int, Tuple[int, int, int]]): Size of the padding applied to the input.
+            stride (Union[int, Tuple[int, int, int]], optional): Stride of the convolution. Default: 1.
+            groups (int, optional): Number of groups in the convolution. Default: 1.
+            norm_layer (Optional[Callable[..., nn.Module]], optional): Normalization layer to use.
+                If None, identity is used. Default: None.
+            activation_layer (Optional[Callable[..., nn.Module]], optional): Activation layer to use.
+                If None, identity is used. Default: None.
+            bias (bool, optional): Whether to use bias in the convolution. Default: False.
+            **kwargs (dict): Additional keyword arguments passed to nn.Conv3d.
+
+        Attributes:
+            kernel_size (Tuple[int, int, int]): Size of the convolution kernel.
+            stride (Tuple[int, int, int]): Stride of the convolution.
+            out_channels (int): Number of output channels.
+        """
+        kernel_size = _triple(kernel_size)
+        stride = _triple(stride)
+        padding = _triple(padding)
+        if norm_layer is None:
+            norm_layer = nn.Identity
+        if activation_layer is None:
+            activation_layer = nn.Identity
+        self.kernel_size = kernel_size
+        self.stride = stride
+
+        dict_layers = OrderedDict(
+            {
+                "conv3d": nn.Conv3d(
+                    in_planes,
+                    out_planes,
+                    kernel_size=kernel_size,
+                    stride=stride,
+                    padding=padding,
+                    groups=groups,
+                    bias=bias,
+                    **kwargs,
+                ),
+                "norm": norm_layer(out_planes, eps=0.001),
+                "act": activation_layer(),
+            },
+        )
+
+        self.out_channels = out_planes
+        super().__init__(dict_layers)
+
+
+class ConvBlock3D(nn.Module):
+    """A module that represents 3D Convolution block."""
+
+    # pylint: disable=too-many-instance-attributes, too-many-arguments
+    def __init__(
+        self,
+        in_planes: int,
+        out_planes: int,
+        kernel_size: tuple[int, int, int],
+        tf_like: bool,
+        conv_type: str | None,
+        padding: tuple[int, int, int] = (0, 0, 0),
+        stride: tuple[int, int, int] = (1, 1, 1),
+        groups: int | None = 1,
+        norm_layer: Callable[..., nn.Module] | None = None,
+        activation_layer: Callable[..., nn.Module] | None = None,
+        bias: bool = False,
+        **kwargs: dict,
+    ) -> None:
+        """Initialization.
+
+        Args:
+            in_planes (int): Number of input channels.
+            out_planes (int): Number of output channels.
+            kernel_size (Tuple[int, int, int]): Size of the convolution kernel.
+            tf_like (bool): Whether to use TensorFlow-like padding and convolution.
```

Comment:
I don't have deep knowledge about the MoviNet model, since it comes from the other repo. However, I added a small description for the MoViNet class. All modules are related to the MoViNet, I think we don't need to more description. https://github.com/openvinotoolkit/training_extensions/pull/2627/commits/7169459c0e1c71d2ee480b906c4533af36004ca9